tests/scripts/all.sh

```diff
@@ -400,6 +400,18 @@ fi
 
 fi # MemSan
 
+msg "build: cmake build from subdirectory"
+cleanup
+mkdir mbedtls_test_build
+cd mbedtls_test_build
+cmake ..
+make
+
+msg "test: cmake build from subdirectory"
+make test
+cd ..
+rm -rf mbedtls_test_build
+
 msg "Done, cleaning up"
 cleanup
```

Comment:
This final cleanup becomes slightly redundant, but I left it in to avoid future issues when more tests are added.